better_uptime/client.py

```diff
@@ -0,0 +1,45 @@
+BU_API_BASE_URL = "https://uptime.betterstack.com/api/v2"
+BU_MAX_PAGES = 25
+
+
+class BetterUptimeClient:
```

Comment:
Siden denne først er en klasse, kunne det blitt nice å gjøre sesjonshåndteringa internt inne klassen her i stedet for å sende den som parameter til alle metodene? Noe a la:
```py
async with BetterUptimeClient.session() as bu:
    monitors = bu.monitors()
    monitor_groups = bu.monitor_groups()
    ...

?

Reply:
Ja, enig - noe i den duren!